_posts/0000-01-02-G-trapp.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"I am so clever that sometimes I don't understand a single word of what I am saying."